### PR TITLE
Wrap and name GetListOfFilesFromData arguments

### PR DESCRIPTION
**Pull request by xtqqczze**
_Contributes to #14171._

### CODE REVIEW

**Comment by xtqqczze** [comment]
`PowerShell-CI-static-analysis` failures tracked by #14195.

File: src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs [hunk 1]
```diff
@@ -2323,13 +2338,16 @@ private IEnumerable<PSModuleInfo> CreateFakeModuleObject(IEnumerable<ModuleSpeci
...
-                    false
-                    /* don't check file existence - don't want to change current behavior without feature team discussion */,
-                    out fileList))
+            if (!GetListOfFilesFromData(
+                data,
+                moduleManifestPath,
+                key: "FileList",
+                manifestProcessingFlags,
+                moduleBase,
+                extension: "",
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs:2347]: I thought we replaced all "" with `string.Empty`. It is not request for the PR.

**Review by iSazonov — APPROVED** [review]

File: src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs (hunk 1 above)
Comment (xtqqczze) [reply src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs:2347]: Maybe we can use [RCS1078i](https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1078i.md) analyzer.

Comment (xtqqczze) [reply src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs:2347]: I will submit PR for [SA1122](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1122.md).

**Pull request merged**